src/prpy/planning/tsr.py

```diff
@@ -78,25 +97,29 @@ def PlanToTSR(self, robot, tsrchains, tsr_timeout=2.0,
         @param robot the robot whose active manipulator will be used
         @param tsrchains a list of TSR chains that define a goal set
         @param num_attempts the maximum number of planning attempts to make
+        @param num_candidates the number of candidates to consider per chunk
         @param chunk_size the number of sampled goals to use per planning call
         @param tsr_timeout the maximum time to spend sampling goal TSR chains
         @param ranker an IK ranking function to use over the IK solutions
         @param max_deviation the maximum per-joint deviation from current pose
                              that can be considered a valid sample.
         @return traj a trajectory that satisfies the specified TSR chains
         """
+        if num_samples is not None:
+            warnings.warn('num_samples is deprecated and has no effect.',
+                DeprecationWarning)
+
```

Comment:
I don't think `num_samples` ever existed in master. It only existed in `bugfix/tsr` branch, which should be deleted once we merge this into master. I suggest remove this parameter.